cli/ts/commands/poll.ts

```diff
@@ -40,12 +40,12 @@ export const getPoll = async ({
 
   const pollContract = PollFactory.connect(pollAddress, signer ?? provider);
 
-  const [[deployTime, duration], isStateAqMerged] = await Promise.all([
+  const [[deployTime, duration], mergedStateRoot] = await Promise.all([
     pollContract.getDeployTimeAndDuration(),
-    pollContract.stateMerged(),
+    pollContract.mergedStateRoot(),
   ]);
 
-  const numSignups = await (isStateAqMerged ? pollContract.numSignups() : maciContract.numSignUps());
+  const numSignups = await (mergedStateRoot ? pollContract.numSignups() : maciContract.numSignUps());
```

Comment:
does this work as expected given the default value for when it's not set it's 0?

Reply:
> does this work as expected given the default value for when it's not set it's 0?

Ran the test suit and didn't revert, could be that we aren't testing that, we can be sure by adding a check that if it's not equal to 0